src/components/experiences/initial-experiences.js

```diff
@@ -33,11 +33,11 @@ export default function InitialExperiences(props) {
     </Content>
      <Actions>
      <RateButton color={'primary'} variant={isPosSelected(true)} onClick={() => handleFeedback(true)}> 
-       <ThumbUpIcon style={{margin: '5%'}} fontSize={'large'}/> 
+       <ThumbUpIcon style={{margin: '5%'}} fontSize={window.innerWidth < 960 ? 'medium' : 'large'}/> 
```

Comment:
same

Reply:
ternaries are now wrapped in parenthesis